Dockerfile-dev

```diff
@@ -0,0 +1,9 @@
+# Base
+FROM alpine:3.7
+RUN apk add --no-cache nodejs nodejs-npm tini
+VOLUME ["/root/app"]
+WORKDIR /root/app
+ENTRYPOINT ["/sbin/tini", "--"]
+
+EXPOSE 3000
+CMD ["sh", "-c", "npm set progress=false && npm config set depth 0 && npm install && npm run start"]
```

Comment:
if you install nodemon and use `nodemon www/bin` instead of `npm run start` this will auto restart the app when any file is edited. Could be added here or in the start script.

Reply:
Thanks, indeed very useful tool.

Update pushed.

Reply:
Reverting, as nodemon is already backed in.